lib/stimulus_reflex/reflex.rb

```diff
@@ -91,7 +77,7 @@ def request
       req = ActionDispatch::Request.new(env)
 
       # fetch path params (controller, action, ...) and apply them
-      request_params = StimulusReflex::RequestParameters.new(params: @params, req: req, url: url)
+      request_params = StimulusReflex::RequestParameters.new(params: form_params, req: req, url: url)
```

Comment:
I'm pretty sure this should be `params: data.params` now instead of `params: form_params` and I think the delegation to `form_params` in the class definition is not needed.